samples/05.c.presentation-mode-styling/index.html

```diff
@@ -30,8 +30,17 @@
 
         const res = await fetch('https://webchat-mockbot.azurewebsites.net/directline/token', { method: 'POST' });
         const { token } = await res.json();
+        const store = window.WebChat.createStore({}, ({ dispatch }) => next => action => {
+          // The following code is for convenient when trying out this sample, not required for production.
```

Comment:
Can this comment be written like this?

`The following code is for convenience only. It not required for production.`